docs/guides/prisma.md

```diff
@@ -0,0 +1,360 @@
+
+You can then apply the generated migration using the Platformatic DB CLI:
+
+```bash
+npx platformatic db migrations apply
+```
+
+Platformatic uses [Postgrator](https://www.npmjs.com/package/postgrator) to run migrations. Postgrator creates a table in the database called `versions` to track the applied migrations. Since the `versions` table is not yet captured in the Prisma schema, run the following command to introspect the database and populate it with the missing model:
+
+```
+npx prisma db pull
+```
+
+Introspecting the database to populate the model prevents including the `versions` table in the generated down migrations.
+
+Your Prisma schema should now contain the `versions` model as follows:
+
+```diff
+model Post {
+  id        Int      @id @default(autoincrement())
+  title     String
+  content   String?
+  published Boolean  @default(false)
+  viewCount Int      @default(0)
+  createdAt DateTime @default(now())
+}
+
++model versions {
++  version BigInt    @id
++  name    String?
++  md5     String?
++  run_at  DateTime? @db.Timestamptz(6)
++}
+```
+
+
+Add the `@@ignore` attribute function to the model to exclude it from the Prisma Client API:
+
+```diff
+model versions {
+  version BigInt    @id
+  name    String?
+  md5     String?
+  run_at  DateTime? @db.Timestamptz(6)
+
++  @@ignore
+}
+```
+### Evolving your database schema
+
+Update the data model in your Prisma schema by adding a model or a field:
+
+```diff
+// based on the schema in the "Adding Prisma to a new project" section
++model User {
++  id    Int     @id @default(autoincrement())
++  email String  @unique
++  name  String?
++  posts Post[]
++}
+
+model Post {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  title     String
+  content   String?
+  published Boolean  @default(false)
+  viewCount Int      @default(0)
++  author    User?    @relation(fields: [authorId], references: [id])
++  authorId  Int?
+}
+
+model versions {
+  version BigInt    @id
+  name    String?
+  md5     String?
+  run_at  DateTime? @db.Timestamptz(6)
+
+  @@ignore
+}
+```
+
+Next, use the `@ruheni/db-diff` CLI tool to generate `up` and `down` migrations:
+
+```bash
+npx db-diff
+```
+
+This command will generate up and down migrations based off of your Prisma schema. If you are currently using a different path to store the migration, you can provide the `--migrations-dir` flag followed by the path.
+
+Next, apply the generated migration using the Platformatic CLI:
+
+```bash
+npx platformatic db migrations apply
+```
+
+And you're done!
+
+## Extending functionality with Plugins using Prisma Client
+
+:::danger
+The following functionality is only supported on Node v18 
+:::
+
+You can use Prisma Client to 
+```bash
+npx prisma generate
+```
+
+The above command will generate Prisma Client based off your Prisma schema.
+
+```js
```

Comment:
This code block could use an introductory sentence explaining that it's a Fastify plugin and should be added to a plugin script that is configured in your Platformatic app configuration.

I'd suggest linking to the 'Add Custom Functionality' guide too, using this path: `/docs/guides/add-custom-functionality/introduction.md` (it works on GitHub and for the rendered docs in Docusaurus).